scripts/send_digest.py

```diff
@@ -0,0 +1,113 @@
+"""Script for sending OSF email digests to subscribed users and removing the records once sent."""
+
+import logging
+import datetime
+from bson.code import Code
+from framework.tasks import app as celery_app
+from modularodm import Q
+from modularodm.exceptions import NoResultsFound
+from framework import sentry
+from framework.auth.core import User
+from framework.mongo import database as db
+from website import mails
+from website.util import web_url_for
+from website.app import init_app
+from website.notifications.model import DigestNotification
+from website.notifications.utils import NotificationsDict
+
+from scripts import utils as script_utils
+
+logger = logging.getLogger(__name__)
+logging.basicConfig(level=logging.DEBUG)
+
+
+def main():
+    script_utils.add_file_logger(logger, __file__)
+    app = init_app()
+    celery_app.main = 'scripts.send_digest'
```

Comment:
I've named the celery_app in `scripts.send_digest.py` but I think it should be set back to `None` after the script executes so as not to interfere with other celery imports. Would I need to do this in a second callback function? @JeffSpies @sloria